File: week2/individuals_js/4.js
function getNumber(index) {
    let numList = [];

    for (let i = 0; i < index * 4; i++) {
        if (i % 7 == 0) {
            numList.push(i);
            numList.push(i + 4);
            numList.push(i + 8);
        }
    }

    console.log(numList[index]);
}

getNumber(1); // print 4
getNumber(5); // print 15
getNumber(10); // print 25
getNumber(30); // print 70

File: week2/individuals_py/4.py

# 0, 4, 8,
# 7, 11, 15,
# 14, 18, 22,
# 21, 25, ...

# 0               + 4 + 4
# 0 + 7           + 4 + 4
# 0 + 7 + 7       + 4 + 4


def get_number(index):
    # your code here
    num_list = []

    for i in range(index*4):
        if i % 7 == 0:
            num_list.append(i)
            num_list.append(i + 4)
            num_list.append(i + 8)
    
    print(num_list[index])


get_number(1) # print 4
get_number(5) # print 15
get_number(10) # print 25
get_number(30) # print 70